test/versioned/mcp-sdk/client.test.cjs

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const test = require('node:test')
```

Comment:
are you intending to also create an esm based file? If not, the extensions in this folder can be `.js`